src/components/forms/forms.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
+import {
+  getFormsDataThunk,
+  deleteFormSchemaThunk,
+  saveFormDataThunk
+} from './../../redux/forms-Reducer';
+import style from './forms.module.css';
+
+function Forms(props) {
+  const [state, setState] = useState({});
+
+  if (!props.checked) {
+    props.getFormsDataThunk();
+  } else if (!state.forms) setState({ forms: [...props.forms] });
+
+  const onSubmit = (e, key) => {
+    e.preventDefault();
+    let data = {};
+    const stateCopy = { ...state };
+    stateCopy.forms[key].inputs.forEach((item, index) => {
+      if (item.value)
+        data[(item.label + key + index).toLowerCase()] = item.value;
```

Comment:
`(item.label + key + index).toLowerCase()` -> this is a /utils.js function, something like: buildUniqueKey, or getUniqueInputKey